mmv1/products/tpuv2/Vm.yaml

```diff
@@ -88,3 +94,220 @@ properties:
     name: 'description'
     description: |
       Text description of the TPU.
+  - !ruby/object:Api::Type::String
+    name: 'cidrBlock'
+    default_from_api: true
+    description: |
+      The CIDR block that the TPU node will use when selecting an IP address. This CIDR block must
+      be a /29 block; the Compute Engine networks API forbids a smaller block, and using a larger
+      block would be wasteful (a node can only consume one IP address). Errors will occur if the
+      CIDR block has already been used for a currently existing TPU node, the CIDR block conflicts
+      with any subnetworks in the user's provided network, or the provided network is peered with
+      another network that is using that CIDR block.
+  - !ruby/object:Api::Type::NestedObject
+    name: 'networkConfig'
+    default_from_api: true
+    immutable: true
+    description: |
+      Network configurations for the TPU node.
+    properties:
+      - !ruby/object:Api::Type::String
+        name: 'network'
```

Comment:
I'm not sure, I can check

Reply:
Ok, I confirmed that you can specify `subnetwork` without `network`. Is there a config change you were thinking of that would capture that?

Reply:
Assuming you tested with a non-default subnetwork we may want to change the docs, since I'm assuming the field is either redundant or defaults to the network of the subnetwork in that case.

Reply:
Ah yea, I see some other resources have better descriptions, which appear to better match the behavior I'm seeing. What if we changed to:
- network: "The name of the network for the TPU node. It must be a preexisting Google Compute Engine network. If both network and subnetwork are specified, the given subnetwork must belong to the given network. If network is not specified, it will be looked up from the subnetwork if one is provided, or otherwise use "default"."
- subnetwork: "The name of the subnetwork for the TPU node. It must be a preexisting Google Compute Engine subnetwork. If both network and subnetwork are specified, the given subnetwork must belong to the given network. If subnetwork is not specified, the subnetwork with the same name as the network will be used."

Reply:
Assuming that's consistent with the behaviour of the resource, yep!

Reply:
It is! I tested out the scenarios described to confirm.